onnxruntime/core/flatbuffers/schema/ort.fbs.h

```diff
@@ -279,7 +279,7 @@ inline const char * const *EnumNamesTensorDataType() {
 }
 
 inline const char *EnumNameTensorDataType(TensorDataType e) {
-  if (flatbuffers::IsOutRange(e, TensorDataType::UNDEFINED, TensorDataType::BFLOAT16)) return "";
+  if (flatbuffers::IsOutRange(e, TensorDataType::UNDEFINED, TensorDataType::FLOAT8E5M2FNUZ)) return "";
```

Comment:
This change was auto generated by the flatbuffer schema. @xadupre this maybe of interest to you.